post_introspect.yml

```diff
@@ -119,11 +118,23 @@
       - name: set fact for storage nodes
         set_fact:
           storage_node_disks: "{{ hostvars['undercloud']['storage_node_disks']|unique }}"
-        when: ceph_enabled and storage_node_disks is not defined
+        when: ceph_enabled and storage_node_disks is not defined and osp_release|int < 17
 
       - name: generate internal.yml.j2
         template:
           src: internal.yml.j2
           dest: ~/.infrared/plugins/tripleo-overcloud/templates/storage/internal.yml.j2
           force: yes
-        when: ceph_enabled
+        when: ceph_enabled and osp_release|int < 17
+
+      - name: generate osd-spec.yml.j2
+        template:
+          src: osd-spec.yml.j2
+          dest: "{{ ansible_user_dir }}/osd-spec.yml"
+          force: yes
+        when: ceph_enabled and osp_release|int == 17
```

Comment:
I am still seeing `==17` in some places. Can you change it to `>=17` wherever it is `==17` in this PR?